src/reader/reader.js

```diff
@@ -0,0 +1,20 @@
+import JSZip from 'jszip';
+import { parseDiffractogram } from '../parser/parser';
+
+/**
+ * @param  {} binary
+ * @param  {} options={}
+ */
+// eslint-disable-next-line no-unused-vars
+export function readBRML(binary, options = {}) {
```

Comment:
Please use rather async / await syntax. We find the code easier to read. It looks like sync code while still being async.

Reply:
Hopefully a bit clearer in https://github.com/cheminfo/xrd/pull/3/commits/da0378de81d8d6650c1fed400834b62e3722d7f7.